config/routes.rb

```diff
@@ -33,6 +33,8 @@
       resources :wishes, only: [:new, :create, :index]
     end
 
+    resources :wishes, only: [:create]
+
```

Comment:
Why extract the create method from the nesting of wishlists? If we do so, do we need to keep both?